src/pages/components/card-section.mdx

```diff
@@ -5,39 +5,45 @@ description: Card section is a collection of cards presented in a full-width sec
 
 import ComponentDescription from "components/ComponentDescription";
 import ComponentFooter from "components/ComponentFooter";
+import sketchSymbol from '../../images/sketch-symbol.svg';
+import githubSymbol from '../../images/icon/github-icon.svg';
+import reactSymbol from '../../images/react-icon.svg';
+import webComponentsSymbol from '../../images/web-components-icon.svg';
 import ResourceLinks from "components/ResourceLinks";
 
 <ComponentDescription name="Card section" type="layout" />
 
 <AnchorLinks>
-
-<AnchorLink>Resources</AnchorLink>
 <AnchorLink>Overview</AnchorLink>
-<AnchorLink>Variations</AnchorLink>
-<AnchorLink>Content guidance</AnchorLink>
+<AnchorLink>Card section</AnchorLink>
+<AnchorLink>Card section with images</AnchorLink>
 <AnchorLink>Feedback</AnchorLink>
-
 </AnchorLinks>
 
-<ResourceLinks name="Card section" type="layout" />
-
 ## Overview
 
-Card section is a collection of text-based cards presented in a full-width section with a left-column header. This pattern is typically used for presenting resources or links.
+Card section is a collection of text-based cards that can include images instead of body copy.
 
-![A visual of Card section simple](../../images/pattern/card-section/card-section-simple.jpg)
+## Card section
+
+Card section uses uses cards with a header and body copy to present links to other parts of IBM.com. 
 
-<br />
+![A visual of Card section simple](../../images/pattern/card-section/card-section-simple.jpg)
+<Caption>An example of Card section</Caption>
 
-## Variations
+### Modifiers
 
-### With CTA
+#### With CTA
 
 In this variation, the last card in the Card group is set to high contrast and can be used to highlight the primary action that the user may take.
 
 ![A visual of Card section with CTA](../../images/pattern/card-section/card-section-cta.jpg)
+<Caption>When Card section has a CTA, the last card will be set to high contrast</Caption>
+
 
-## Content guidance
+<ResourceLinks name="Card section" type="layout" />
```

Comment:
@RichKummer, you'll need to add the multi prop to the `Resources` for `Card section` because it's currently showing as an H2 and it doesn't have the full heading name.